src/handy.jl

```diff
@@ -35,13 +35,13 @@ function mysql_connect(host::AbstractString,
 end
 
 """
-    mysql_connect(host::AbstractString, user::AbstractString, passwd::AbstractString, db::AbstractString = ""; opts = Dict())
+    mysql_connect(host::AbstractString, user::AbstractString, passwd::AbstractString, db::AbstractString = ""; port::Int64 = 3306, socket::AbstractString = "/var/run/mysqld/mysqld.sock", opts = Dict())
 
 Connect to a MySQL database.
 """
-function mysql_connect(host, user, passwd, db=""; opts = Dict())
-    return mysql_connect(host, user, passwd, db, convert(Cuint, 0),
-                         convert(Ptr{Cchar}, C_NULL), CLIENT_MULTI_STATEMENTS, opts=opts)
+function mysql_connect(host, user, passwd, db=""; port=3306, socket="/var/run/mysqld/mysqld.sock", opts = Dict())
```

Comment:
The default value for socket should be a constant defined in consts.jl and the right value set for different OSes using `is_unix`, `is_windows` etc.